cerebrium/container-images/defining-container-images.mdx

```diff
@@ -0,0 +1,355 @@
+---
+title: Defining Container Images
+---
+
+## Introduction
+
+Cerebrium simplifies the deployment and running of machine learning apps by abstracting infrastructure management into configuration, allowing engineering teams to focus on what matters most - delivering value to customers using application code. A single TOML file manages environment setup, deployments, and scaling - tasks that typically require dedicated teams.
+
+## Why TOML?
+
+While Python decorators offer programmatic configuration, they scatter infrastructure settings throughout code files, making changes risky and review difficult. TOML centralizes all configuration in one place, making it easier to track changes and maintain consistency. Its straightforward syntax prevents the accidental complexity that often comes with code-based configuration, while its hierarchical structure naturally maps to modern app requirements.
+
+## Understanding containers on Cerebrium
+
+Cerebrium handles containers differently from traditional Docker or Kubernetes setups. Instead of managing multiple configuration files and orchestration rules, developers declare their requirements in `cerebrium.toml`. The system automatically handles container lifecycle, networking, and scaling based on this configuration.
+
+### Getting Started
+
+The fastest and simplest way to create a config file is to run the **`cerebrium init`** command. This command creates a **`cerebrium.toml`** file in the project root, which can then be edited to suit specific application requirements.
```

Comment:
after reading this doc, it isn't clear how the default runtime (cortex) wants the project directory to be structured (ie, you need a `main.py` entrypoint, the app is run from the `cortex/` dir, you can have any other dirs in your root, you need to explicitly include/exclude dirs if you are using a venv, etc.)